docs/markdown/GuiTutorial.md

```diff
@@ -95,6 +95,18 @@ going to go into how it works. Merely replace the contents of
 ```c
 #include "SDL.h"
 
+//FIX WINMAIN ISSUE ON COMPILE
```

Comment:
What is the issue with this comment? Also doesn't WinMain() need to be defined after the real main()?